internal/privi/pds_forwarding.go

```diff
@@ -12,22 +12,27 @@ import (
 	"github.com/rs/zerolog/log"
 )
 
-type pdsForwarding struct {
+type PDSForwarding struct {
 	oauthServer *oauthserver.OAuthServer
 	dir         identity.Directory
 }
 
-var _ http.Handler = (*pdsForwarding)(nil)
+var _ http.Handler = (*PDSForwarding)(nil)
 
-func newPDSForwarding(oauthServer *oauthserver.OAuthServer) *pdsForwarding {
-	return &pdsForwarding{
+func NewPDSForwarding(oauthServer *oauthserver.OAuthServer) *PDSForwarding {
+	return &PDSForwarding{
 		oauthServer: oauthServer,
 		dir:         identity.DefaultDirectory(),
 	}
 }
 
 // ServeHTTP implements http.Handler.
-func (p *pdsForwarding) ServeHTTP(w http.ResponseWriter, r *http.Request) {
+func (p *PDSForwarding) ServeHTTP(w http.ResponseWriter, r *http.Request) {
```

Comment:
what's the purpose for this change? i like it better inline

Reply:
The next PR needs to be able to query the bluesky PDS with an arbitrary endpoint, but still forward the credentials from the request